lib/src/main/java/uk/gov/di/ipv/core/library/service/CiMitService.java

```diff
@@ -149,6 +149,23 @@ public List<ContraIndicatorItem> getCIs(
     public ContraIndications getContraIndicatorsVC(
             String userId, String govukSigninJourneyId, String ipAddress)
             throws CiRetrievalException {
+        SignedJWT ciSignedJWT = getContraIndicatorsJWT(userId, govukSigninJourneyId, ipAddress);
+        ContraIndicatorEvidenceDto contraIndicatorEvidence =
+                parseContraIndicatorEvidence(ciSignedJWT);
+
+        return mapToContraIndications(contraIndicatorEvidence);
+    }
+
+    public String getContraIndicatorsVcAsJwtString(
```

Comment:
Or just make `getContraIndicatorsJWT` (which I'd rename to `getContraIndicatorsVCJwt`) public and let BuildUserIdentity serialise it to string?

Reply:
Done.